Recipes/Recipes/ProfileModule/ViewControllers/BonusesViewController.swift

```diff
@@ -0,0 +1,141 @@
+// BonusesViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+private enum Constants {
```

Comment:
```suggestion
private enum Constants {
```
в констролер везде такое перенеси